scalding-core/src/test/scala/com/twitter/scalding/typed/OptimizationRulesTest.scala

```diff
@@ -30,7 +29,7 @@ object TypedPipeGen {
         tpGen(srcGen).map { p: TypedPipe[Int] =>
           { x: TypedPipe[Int] => x.cross(ValuePipe(2)).values }
         },
-        Gen.const({ t: TypedPipe[Int] => t.debug }),
+        //Gen.const({ t: TypedPipe[Int] => t.debug }), debug spews a lot to the terminal
```

Comment:
should we leave this commented out/enabled ? (just don't know if we have any concerns over debug's correctness, we won't test it like this. But that might be fine)

Reply:
We almost never use debug in real life and it was making test logs unreadable.